packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -102,10 +105,15 @@ async function orchestratorProcess(
         undefined,
         args.browserAuth);
 
-    // Create a new file if a testId wasn't provided
-    const url = args.testId !== undefined
-        ? await testDriver.createContainerUrl(args.testId)
-        : await initialize(testDriver, seed, profile, args.verbose === true);
+    let url;
+    if (args.testId !== undefined && args.createTestId === false) {
+        // If testId is provided and createTestId is false, then load the file;
+        url = await testDriver.createContainerUrl(args.testId);
+    } else {
+        // If no testId is provided, (or) if testId is provided but createTestId is not false, then create a file;
+        // In case testId is provided, name of the file to be created is taken as the testId provided
+        url = await initialize(testDriver, seed, profile, args.verbose === true, args.testId);
+    }
```

Comment:
>  it will end up creating a new document with the id the second time as well

(Letting my unfamiliarity with this show 😄 ): Is that possible? I expected doc ids to be unique.

I was thinking of the scenario where tests ran, some passed, some failed, and someone retried the job because the failures might have been transient, or just flaky tests. And what would happen to the tests that created docs with hardcoded ids on the first pass.

Reply:
most of the time we never set test id, we just use a random file name. test id i only currently used to connect to a specific file, with the create flag it will also create a specific file. this seems fine.

Reply:
please update the readme as well